pom.xml

```diff
@@ -226,11 +212,12 @@
             <groupId>org.seleniumhq.selenium</groupId>
             <artifactId>htmlunit-driver</artifactId>
         </dependency>
-<!--        <dependency>--> <!-- Used to generate OpenAPI docs -->
-<!--            <groupId>org.springdoc</groupId>-->
-<!--            <artifactId>springdoc-openapi-webmvc-core</artifactId>-->
-<!--            <version>1.6.12</version>-->
-<!--        </dependency>-->
+        <!-- doc -->
+        <dependency>
+            <groupId>org.springdoc</groupId>
+            <artifactId>springdoc-openapi-starter-webmvc-ui</artifactId>
```

Comment:
What is this used for?

Reply:
springdoc-openapi is a spring boot plugin for generate the open-api 3.0 and provide the swager-UI
see https://springdoc.org/